server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -1533,14 +1528,35 @@ public TSStatus createMultiTimeseries(TSCreateMultiTimeseriesReq req) {
             req.getPaths().size(), req.getPaths().get(0));
       }
       List<TSStatus> statusList = new ArrayList<>(req.paths.size());
+      CreateTimeSeriesPlan plan = new CreateTimeSeriesPlan();
```

Comment:
What is this plan for?

Reply:
Just to do some check, like checkAuthority before execute the CreateMultiTimeSeriesPlan.